gcp_variant_transforms/testing/integration/run_preprocessor_tests.py

```diff
@@ -71,8 +70,7 @@ def __init__(self,
     self._report_blob_name = self._append_suffix(report_blob_name, suffix)
     self._report_path = '/'.join(['gs:/', _BUCKET_NAME, self._report_blob_name])
     self._project = parser_args.project
-    args = ['--input_pattern {}'.format(input_pattern),
-            '--report_path {}'.format(self._report_path),
+    args = ['--report_path {}'.format(self._report_path),
```

Comment:
The input_pattern (or input_file) is require to run the test, Did you provide the input_pattern somewhere else?